content/applications/finance/fiscal_localizations/hong_kong.rst

```diff
@@ -51,5 +61,389 @@ QR-code` option to *EMV Merchant-Presented QR-code*.
+   contribution starts on second month and the **employer** contribution for the second month is
+   include in the first month as back-pay.
+
+Under the :guilabel:`Other Inputs` tab at the bottom of payslip, there are additional manual input
+types that are specific to HK Payroll:
+
+- :guilabel:`Back Pay`: Additional salary payout can be included under this category.
+- :guilabel:`Commission`: The commission earned during the period can be manually entered here.
+- :guilabel:`Global Deduction`: A lump-sum deduction from the entire payslip.
+- :guilabel:`Global Reimbursement`: A lump-sum reimbursement to the entire payslip.
+- :guilabel:`Referral Fee`: The additional bonus offered for any form of business-related referral.
+- :guilabel:`Moving Daily Wage`: To override the :abbr:`ADW (Average Daily Wage)` value used for leaves computation.
+- :guilabel:`Skip Rent Allowance`: If set, the rental allowance is excluded from the current payslip.
+- :guilabel:`Custom Average Monthly Salary`: To override the average monthly salary used for end-of-year payment.
+
+Once the payslips are ready, click :guilabel:`Create Draft entry` to generate a draft journal entry 
+found in the :guilabel:`Other Info` tab of the payslip.
+
+Paying employees
+----------------
+
+Once the draft journal entries have been posted, the company can now pay the employees.
+The user can choose between **two** different **payment methods**.
+
+- From the employee's payslip (:menuselection:`Payroll app --> Payslips`), once the payslip's journal
+  entry has been posted, click :guilabel:`Register Payment`. The process is the same as
+  :doc:`paying vendor bills <../accounting/payments>`: select the desired bank journal and payment
+  method, then later reconcile the payment with the corresponding bank statement.
+
+- For batch payments, once  all draft journal entries from the batch are confirmed, click :guilabel:`Mark as Paid` 
+  to post the payment journal entry. Then create a manual payment in the *Accounting* app and 
+  reconcile accordingly.
+
+Attendances & Hourly Wage
+-------------------------
+
+Setup the contract as follows for employees who are based on hourly-wage contract:
+
+.. important::
+   Make sure the employee contract is using **Attendance** as the Work Entry Source and the Wage
+   Type is set to **Hourly Wage**.
+
+#. Go to *Attendance* app.
+#. The employee can check-in/out via the kiosk mode.
+#. In the *Payroll* app, review the attendance work entries generated from
+   :menuselection:`Payroll app --> Work Entries`.
+#. Next, generate the :ref:`payslips <hong_kong/running_payslips>` and process the payment.
+
+.. image:: hong_kong/hk-attendance-work-entry.png
+   :alt: Hong Kong Attendance Work Entry
+
+.. image:: hong_kong/hk-attendance-payslip.png
+   :alt: Hong Kong Attendance Payslip
+
+Time-Off with Payroll
+---------------------
+
+The work entry types and time-off types are fully integrated between the *Time-off* and
```

Comment:
```suggestion
The work entry types and time off types are fully integrated between the *Time-off* and
```
The app is called Time Off and I believe all reference do not have a hyphen. Please double check this is correct- always ave it witten as it does in the specific screen shot for the step. However it is, it needs to be corrected in the below as well.